### DISCUSSION TITLE
Morphing Card - action buttons do not appear correctly in circle and hexagon

### DISCUSSION THREAD

**Discussion by luiztux** [nikhils4/ui-beats#15] [General]
Guys, the components are simply fantastic. I'm loving them.

That said, I didn't want to open an issue because I don't think it's necessary, it would be more to discuss the positioning of the action buttons on the Morphing Card.

On the website you can see that, when using the circle and hexagon shapes, the buttons almost disappear, staying "outside" the border.

- square (ok)

![image](https://github.com/user-attachments/assets/fce8ffc6-b775-4696-ae85-497dcd513233)

- circle

![image](https://github.com/user-attachments/assets/624d74aa-d356-4125-bbfd-9efa3fd1479d)

- hexagon

![image](https://github.com/user-attachments/assets/4e033340-bd16-426b-975a-688d8d60a04f)

**Comment by nikhils4** [comment]
Thank you so much @luiztux, i will definitely have a look and open to your suggestions on how can we improve this 😀

**Reply by nikhils4** [reply]
Inviting @lappemic to this discussion, if he has something to add on 💯

**Reply by lappemic** [reply]
Really sorry nikhil! I missed this! Please ping me again if you think i can add value at some point.

**Reply by nikhils4** [reply]
No worries at all @lappemic 😄 

@mr-joshi fixed it already

**Comment by nikhils4** [comment]
@mr-joshi has fixed this issue with #17 

Closing this!

**Reply by mr-joshi** [reply]
😀
